### PR TITLE
teams/docs: a vote for en-jin19 as the committer

### PR DESCRIPTION
**Pull request by qiancai**
As stated in the title.

Hi https://github.com/orgs/pingcap/teams/tidb-maintainers, PTAL.

/hold

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM
Congratulations!

**Review by lilin90 — APPROVED** [review]
LGTM

File: votes/0634-en-jin19-as-docs-committer.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
+
+(To be updated) Conclude the voting result, including approvals and vetoes, binding and non-binding.
```
Comment (qiancai) [inline votes/0634-en-jin19-as-docs-committer.md:35]:
```suggestion
Approved by 3 binding votes:

* lilin90 (binding)
* qiancai (binding)
* TomShawn (binding)
```

**Pull request merged**